docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.8'
+
+services:
+  db:
+    image: mysql:8.0
+    container_name: mysql_db
+    environment:
+      MYSQL_ROOT_PASSWORD: root_password
+      MYSQL_DATABASE: todolist_db
+      MYSQL_USER: user
+      MYSQL_PASSWORD: password
```

Comment:
Using hardcoded credentials like `root_password`, `user`, and `password` is not secure. Consider using environment variables or Docker secrets to manage sensitive information securely.